docs/api.rst

```diff
@@ -0,0 +1,7 @@
+API Reference
+=============
+
+.. toctree::
+   :maxdepth: 2
+
+   api/nameko
```

Comment:
yeah let's drop the whole thing. we can add it back eventually with more prose around the various sections